### PR TITLE
#13667 - SBC-AUTH - Account Mailer - Update email templates with optimized font files

### PR DESCRIPTION
**Pull request by MatthewCai2002**
*Issue #: 13667*
https://github.com/bcgov/entity/issues/13667

*Description of changes:*
    - removed extra font files from account_mailer
    - updated pdf-template fonts with optimized fonts 
    - updated email template fonts with optimized fonts

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: queue_services/account-mailer/src/account_mailer/email_templates/common/style.html [hunk 1]
```diff
@@ -4,8 +4,7 @@
...
...WxJ36VVPOX4gBLREd8eWUXRaepsl5Fpai0FyrGp56hxtWr1k9azenrCk8fVYYz8npwEalOyCi1pLl29US29XVEuGsqwprE8yEhg8Stb/mt6FpdfR17R2hbbevue7tkLIzUSdFV8eIys9t+HgIwlsw195dPKrcpRzlihvKCv5QYtN/XkkdesvoFSRS2LZkNXp6zPUTrNXl6+6N8ywVbWd8/VVy20iAh/OUU34+dc2aFM+Fb/AtbqTP0AAAB42m3QR0yTcRjH8e8DpYWy98a9V9+3LcPdAq97b3GhQFtFwGJVXGjcMxoTPWlcFzXuGY16UONGjSPqwbM7HtSrFt6/N5/LJ8+TPE+e/Iigrf64qOF/9RkkQiKJxEIUVmxEE4OdWOKIJ4FEkkgmhVTSSCeDTLLIJodc8singHa0pwMd6URnutCVbnSnBz3pRW/60Jd+ONDQceLCTSFFFFNCfwYwkEEMZghD8eCllDLKMRjGcEYwklGMZgxjGcd4JjCRSUxmClOZxnRmMJMKZjGbOcxlHpVi4Rgb2cRN9vORzexmBwc5wXGJYjvv2cA+sYqNXRxgK3f4INEc4iS/+MlvjnKah9znDPNZwB6qeEw1D3jEM57wlBY+hdN7yXNecBYfP9jLG17xGj9f+MY2FhJgEYuppY7D1LOEBoI0EmIpy1geTnkFK2liFWtYzTWO0Mxa1rGer3znOuc4zw3e8k5ixC6xEifxkiCJkiTJkiKpkibpksEFLnKFq9zlEpe5xxZOSSa3uC1Zks1OyZFcyZN8KbD6apsa/JotVBdwOBxlph6HUvVeXelUlrSqhxeUmlJXOpUupVtZqCxSFiv/3fOYauquptlrAr5QsLqqstFvjnTD1G1YykPB+rbGbZS2anjNP8LqSqfS9Rd66Zz9AAAAeNpFzU0OwVAUBeC+Vv9Q+ouJpExI3grEVDsxEYmkTSQ2YMyQSYes5dZIxIpsglOe19n9Ts7NubN3QeyiLMlaZSVj17xMDZ6NyM2XFK5xnPMhGXybKaTFCWl8Qa04uWkvlX/RBFqFgA00dwIWYKcCJmDNBAzAjAV0wAgEGoC+EGhXO/4PjByxHiB1niovtfQA+mCwl/RAfyrpxslD8dhckUkXBXco2QG7nmRY9Z3Tu+5HVRKycZ308BIdJftgbyM5APuTP3MK+QcE5WOG) format('woff');
```
Comment (seeker25) [inline queue_services/account-mailer/src/account_mailer/email_templates/common/style.html:7]: Didn't we try this one?

Comment (MatthewCai2002) [reply queue_services/account-mailer/src/account_mailer/email_templates/common/style.html:7]: yeah, but I was thinking that it was possibly the extra font files in account-mailer that were actually making the emails larger so I wanted to test if just removing those would fix the problem.

Comment (thorwolpert) [reply queue_services/account-mailer/src/account_mailer/email_templates/common/style.html:7]: The problem is it's not a font published any cached CMS, so folks will get aliased fonts when viewing online.
I'm not sure why we use it tbh, for printing we had to change the font as the gov't print services have issues with it.

Comment (thorwolpert) [reply queue_services/account-mailer/src/account_mailer/email_templates/common/style.html:7]: The other option is just to install the font on the print server and ref it, and let the font aliasing issue be another issue to take care of.

**Comment by seeker25** [comment]
Merging, this isn't going to main

**Pull request merged**